src/content/docs/security/security-privacy/compliance/regulatory-audits-new-relic-services.mdx

```diff
@@ -361,7 +361,7 @@ New Relic's time frames for supported regulatory frameworks and annual audits in
 
     <tr>
       <td>
-        Trace API
+        [Trace API](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints/#trace-api)
```

Comment:
On my local instance, I couldn't load the URL with the anchor; it would only go to the top of the page: /docs/security/security-privacy/compliance/fedramp-compliant-endpoints/#trace-api

I eventually would get this error: Unhandled Runtime Error. You might try this out on your box.

Reply:
that's weird. I can't imagine why it wouldn't work. it's working on public docs.